sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_funnel_v1/checks.sql

```diff
@@ -3,29 +3,29 @@
 -- min_row_count helps us detect if we're seeing delays in the data arriving
 -- could also be an indicator of an upstream issue.
 #fail
-{{ min_row_count(1, "`date` = DATE_SUB(@submission_date, INTERVAL 1 DAY)") }}
+{{ min_row_count(1, "submission_date = @submission_date") }}
 #fail
-WITH _aua_new_profiles AS (
+WITH fx_ios_count AS (
   SELECT
-    SUM(new_profiles) AS new_profiles,
+    COUNT(*)
```

Comment:
In case the bug of duplicates in the first_seen tables is not resolved or happens again, what do you think about  COUNT(DISTINCT) here for the correct number?

Reply:
We have a check on the firefox_ios_clients table to check for duplicates which is set to fail. I think this should be safe to do.
https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/firefox_ios_derived/firefox_ios_clients_v1/checks.sql#L2